gnomic/utils.py

```diff
@@ -1,7 +1,81 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 import collections
-from gnomic.models import Mutation, Plasmid, Fusion, FeatureTree, FeatureSet, Feature
+import html
```

Comment:
Python 2.7 does not have the html module. Need to have a fallback here. A typical approach is something like

```
try:
    import html
    html_escape = html.escape
except ImportError:
    html_escape = ... 

```